src/sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
+    return a + b
+}
+module.exports = sum
```

Comment:
Thanks this is good to know. Since these files are only temporary though I'll be deleting them once we have real tests that we can execute. I will make sure to use only .ts from now on.